src/js/editor/editor.js

```diff
@@ -648,7 +622,7 @@ class Editor {
 
   insertSectionAtCursor(newSection) {
     let newRenderNode = this._renderTree.buildRenderNode(newSection);
-    let renderNodes = this.getRenderNodesWithCursor();
+    let renderNodes = this.cursor.activeSections.map(s => s.renderNode);
```

Comment:
Yeah, perhaps we can start to rectify these into a unified approach soon. I have been thinking of the `post` as the single source of truth, partly because historically renderNodes for a given postNode did not always exist until, so they require some mental bookkeeping to keep track when the last render was, etc. The object model of posts->sections->markers->markups helps me to keep everything conceptually straight as well — some edit actions apply only to one instead of the other (ie block-format commands like h1, h2 apply to sections, text format commands like b, i apply to markers), and I like that the code makes that distinction clear.